fabric-networking-api-v1/src/main/java/net/fabricmc/fabric/api/networking/v1/PlayerLookup.java

```diff
@@ -111,9 +113,15 @@
 
 		if (manager instanceof ServerChunkManager) {
 			ThreadedAnvilChunkStorage storage = ((ServerChunkManager) manager).threadedAnvilChunkStorage;
+			EntityTrackerAccessor tracker = ((ThreadedAnvilChunkStorageAccessor) storage).getEntityTrackers().get(entity.getEntityId());
 
 			// return an immutable collection to guard against accidental removals.
-			return Collections.unmodifiableCollection(((ThreadedAnvilChunkStorageTrackingExtensions) storage).fabric_getTrackingPlayers(entity));
+			if (tracker != null) {
+				return Collections.unmodifiableCollection(tracker.getPlayersTracking()
+						.stream().map(class_5629::method_32311).collect(Collectors.toSet()));
+			}
+
+			return Collections.emptySet();
```

Comment:
Should this be returning an empty set or fail since the entity is not being tracked at all by the server world.

Reply:
This is following the semantics of the late `ThreadedAnvilChunkStorageMixinfabric_getTrackingPlayers`

Reply:
Hmm I see.